### ISSUE TITLE
Do I have to split the training data to five parts to do cross-validation myself?

### ISSUE THREAD

**Issue by wyzjack** [MIC-DKFZ/BraTS2017#7]
Cause I only have training data and validation data(not labeled so actually is test data) ,so do I have to split the training data to five parts to do cross-validation myself or your codes have realized this process?

**Comment by FabianIsensee** [comment]
If your validation data does not have labels then you do the split on your training data. Once the model is trained you can predict it the same way I predicted the BraTS test data

**Comment by wyzjack** [comment]
OK thankyou

**Comment by wyzjack** [comment]
Sorry I am still a little confused that: it seems that your file validation_network.py does the cross-validation process, so I don't need to split the training set myself?

**Comment by poornasandur** [comment]
Hi 
             Can I train the network on a NVIDIA Geforce GTX 970 GPU  4GB and RAM of 8GB ...

**Comment by FabianIsensee** [comment]
No you can't. It needs 12 GB of GPU memory

**Comment by poornasandur** [comment]
Suppose if i reduce the no. of images and then train it batch by batch then in that case can I do it...

**Comment by poornasandur** [comment]
Sir,
      I meant that, I will be taking only half of the HGG data train it and then in the next training I will take the remaining half and then go for prediction. Can I do it that way....

**Comment by FabianIsensee** [comment]
That is not how this works. The number of training cases has no effect on GPU Memory consumption. Batch size, number of filters and patch size are the ones that determine GPU memory usage. My method is designed to use large input patches, so reducing that would not be a good idea (it would defeat the purpose). batch size y 2 (i am using 2 here) is too unstable so the only thing you could do is reduce the number of filters which is a bad idea (they are already quite low in this implementation). I am very sorry, but you cannot use this method with only 4 GB of GPU memory

**Comment by poornasandur** [comment]
oh ok... thnx for the inputs sir...
